test/Microsoft.ML.CpuMath.UnitTests.DotNetCoreApp/UnitTests.cs

```diff
@@ -0,0 +1,451 @@
+using System;
+using System.Runtime.InteropServices;
+using System.Security;
+using Xunit;
+using Xunit.Abstractions;
+using Microsoft.ML.Runtime.Internal.CpuMath;
+
+namespace Microsoft.ML.CpuMath.UnitTests.DotNetCoreApp
+{
+    public class UnitTests
```

Comment:
I think we should give this class a more descriptive name of what it is testing.  Probably something similar to `CpuMathUtilsTests`.

Reply:
Edited thanks